src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -62,6 +62,11 @@ public function getError(Response $response): string
         return $this->getResponseContentValue($response, 'hydra:description');
     }
 
+    public function isAcceptSuccessful(Response $response): bool
```

Comment:
To be 100% grammarly proper, it probably should be `isAcceptanceSuccessful`

Reply:
IMO `isAccepted` would fit best, both `isAcceptSuccessful` and `isAcceptanceSuccessful` don't really make much sense, as we're not requesting the acceptance, its just the result